### DISCUSSION TITLE
CONTEXT MENU

### DISCUSSION THREAD

**Discussion by davidav37** [dyphire/mpv-config#114] [Q&A] [answered]
Good evening 
first of all I would like to congratulate you for the work accomplished, I love this configuration.
I come to you because I translated the entire configuration into French but I have a problem with the context menu, I have a few duplicate menus and I don't see at all where this could come from.
would you have any idea of ​​the problem? THANKS

**Comment by dyphire** [comment]
Please provide your input.conf file and screenshots

**Reply by davidav37** [reply]
![Capture d'écran 2024-08-20 101101](https://github.com/user-attachments/assets/961b5d23-6d55-4573-92de-56ed7ee692d8)
![Capture d'écran 2024-08-20 101248](https://github.com/user-attachments/assets/9e262b66-b1ee-4cea-ac79-3b9f51b348b9)
![Capture d'écran 2024-08-20 101306](https://github.com/user-attachments/assets/a67ebbbd-9816-4875-8cea-b63a207f9d14)
![Capture d'écran 2024-08-20 101320](https://github.com/user-attachments/assets/a224a2ad-ea1d-4988-b68d-03e6478576f6)
[input.txt](https://github.com/user-attachments/files/16671712/input.txt)

**Reply by verygoodlee** [reply, marked as answer]
The reason is you mixed the Space and Non-Breaking Space in menu title, they are two different charaters.
Replace all  Non-Breaking Space by Space should fix it.
[input.txt](https://github.com/user-attachments/files/16672608/input.txt)

**Reply by davidav37** [reply]
thank you my friend the problem is solved 
I would still like to salute the work accomplished here 
well done 
THANKS

**Reply by verygoodlee** [reply]
Most advanced text editors have the option to view all characters, can easily find the reason.

![image](https://github.com/user-attachments/assets/ceb607da-2ef7-4696-9a20-c5f6d59b03fe)

**Reply by dyphire** [reply]
It seems to have been solved. Close.
